test/e2e/networking.go

```diff
@@ -171,7 +171,7 @@ var _ = Describe("Networking", func() {
 				DoRaw()
 		}
 
-		timeout := time.Now().Add(2 * time.Minute)
+		timeout := time.Now().Add(3 * time.Minute)
```

Comment:
please add a comment (sorry to be nitpicky, but someone will surely think this is too large and bring it down and we'll see flake again)